core/cas-server-core-logging/src/main/java/org/apereo/cas/logging/web/ThreadContextMDCServletFilter.java

```diff
@@ -77,6 +77,12 @@ public void doFilter(final ServletRequest servletRequest, final ServletResponse
             request.setAttribute("requestId", requestId);
             response.setHeader("requestId", requestId);
 
+            val sessionId = request.getSession().getId();
+            addContextAttribute("sessionId", sessionId);
+            request.setAttribute("sessionId", sessionId);
+            response.setHeader("sessionId", sessionId);
+
```

Comment:
Why is this setting a new non-standard http header on the response?

Reply:
I replicated the block used to set requestId above, which for whatever reason includes setting a header. Happy to remove it!